client/landing/stepper/declarative-flow/sensei.ts

```diff
@@ -0,0 +1,80 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { SENSEI_FLOW, useFlowProgress } from '@automattic/onboarding';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect } from 'react';
+import { recordFullStoryEvent } from 'calypso/lib/analytics/fullstory';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ONBOARD_STORE, USER_STORE } from '../stores';
+import Launchpad from './internals/steps-repository/launchpad';
+import ProcessingStep from './internals/steps-repository/processing-step';
+import SenseiDomain from './internals/steps-repository/sensei-domain';
+import SenseiLaunch from './internals/steps-repository/sensei-launch';
+import SenseiPlan from './internals/steps-repository/sensei-plan';
+import SenseiSetup from './internals/steps-repository/sensei-setup';
+import { AssertConditionState, Flow } from './internals/types';
+import './sensei.scss';
+
+const sensei: Flow = {
+	name: SENSEI_FLOW,
+	title: 'Sensei',
```

Comment:
You should translate this title

Reply:
Fixed in f3631ed58cff0e73f3f9f714baea43b958b49a1c. Not sure how this will work though. As Sensei is our brand maybe we don't want it translated?

Reply:
Hmm I now see that we also translate Sensei in other files so it should be translated everywhere for consistency.